refinery/core/urls.py

```diff
@@ -11,66 +11,53 @@
 from .views import (AnalysesViewSet, DataSetsViewSet, NodeViewSet, OpenIDToken,
                     WorkflowViewSet)
 
+UUID_RE = r'[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}'
```

Comment:
Good idea but I'd move it somewhere like `refinery/constants.py` (see Two Scoops of Django, p. 38)